lib/android-helpers.js

```diff
@@ -419,7 +419,19 @@ helpers.pushSettingsApp = async function (adb, throwError = false) {
 
 helpers.pushUnlock = async function (adb) {
   logger.debug("Pushing unlock helper app to device...");
-  await adb.installOrUpgrade(unlockApkPath, UNLOCK_HELPER_PKG_ID, true);
+
+  // the unlock helpers does not have the correct versioning, so
+  // `adb.installOrUpgrade` will not work
+  try {
+    await adb.install(unlockApkPath, false);
+  } catch (err) {
+    if (!err.message.includes('INSTALL_FAILED_UPDATE_INCOMPATIBLE')) {
```

Comment:
shouldn't we check if err.message exists first?

Reply:
Sure.